base/setup/welcome/welcome.c

```diff
@@ -210,6 +210,101 @@ VOID TranslateEscapes(IN OUT LPTSTR lpString)
     }
 }
 
+/*
+ * Expands the path for the ReactOS Installer "reactos.exe".
+ * See also base/system/userinit/userinit.c!StartInstaller()
+ */
+VOID ExpandInstallerPath(IN OUT LPTSTR lpInstallerPath, IN SIZE_T PathSize)
```

Comment:
Why LPTSTR?
Is there ever a case where we don't want to build as unicode?

Reply:
I keep the code style of the rest of the app for the moment.